test/test-starts-up-with-data-volume

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset
+
+. "$(dirname "$0")/helpers.sh"
+
+readonly image="$1"
+readonly series="$2"
+readonly cname="neo4j-$(uuidgen)"
+
+readonly datadir=$(mktemp --directory)
```

Comment:
please add `--tmpdir=/tmp/` for OS X compatibility 🍏

Reply:
Ah right. Done